src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw(Figure figure) {
+        System.out.println("Figure: Circle, area: " + getArea() + " sq. units, radius: "
+                + radius + " units, color: " + getColor());
```

Comment:
`color` is a protected field so you can call it immediately, without using the getter